### PR TITLE
APP-672 New Keyboard shortcuts

### PR DESCRIPTION
**Pull request by kvnvelasco**
This change is [Reviewable](https://reviewable.io/reviews/mathspace/msquill/30)

### CODE REVIEW

**Comment by kvnvelasco** [comment]
@bholloway This nit change from the previous set of commits broke keyboard shortcut registration. There's no simple way to detect when this object will init, so checking on registration is the quickest fix.

File: src/commands/mathspace/mathspaceCommands.js [hunk 1]
```diff
@@ -12,6 +12,42 @@ CharCmds['*'] = LatexCmds.times;
 // Patched latex for % symbol, it should not contain \\ in the beginning.
 LatexCmds['%'] = bind(NonSymbolaSymbol, '%', '%');
 
+var MixedFraction =
```
Comment (bholloway) [inline src/commands/mathspace/mathspaceCommands.js]: Ideally I would like to see a definition consistent with what has come before.

**opinion** The additional `var` here is confusing to me.

**question** How is the `P(...)` definition different to the `bind` definition?

Comment (bholloway) [reply src/commands/mathspace/mathspaceCommands.js]: withdrawn: I now see this is consistent with what is already here (below)

File: src/commands/mathspace/mathspaceCommands.js [hunk 2]
```diff
@@ -12,6 +12,42 @@ CharCmds['*'] = LatexCmds.times;
 // Patched latex for % symbol, it should not contain \\ in the beginning.
 LatexCmds['%'] = bind(NonSymbolaSymbol, '%', '%');
 
+var MixedFraction = 
+  LatexCmds.mixed = P(MathCommand, function(_, super_) {
+    _.ctrlSeq = '\\frac';
+    _.htmlTemplate = 
+    '<span>'
```
[the remark this answers is not in the document]
Comment (bholloway) [reply src/commands/mathspace/mathspaceCommands.js]: withdrawn: I now see this is consistent with what is already here (below)

**Pull request merged**